pkg/network/common/common.go

```diff
@@ -88,6 +79,21 @@ func ParseClusterNetwork(cn *networkv1.ClusterNetwork) (*ParsedClusterNetwork, e
 	return pcn, nil
 }
 
+// Contains determines whether pcn contains ip
```

Comment:
nit, I see more readable to have 2 methods instead , i.e
`PodNetworkContains()` and `ServiceNetworkContains()`
 discrimination on the output variables makes the code to use a lot underscores for unused variables